.github/workflows/wpt-mapper.yml

```diff
@@ -50,28 +45,33 @@ jobs:
         working-directory: wpt
       - name: Install Chromium
         run: echo "chromium_binary=$(npx @puppeteer/browsers install chromium@latest | cut -f 2 -d' ')" >> $GITHUB_ENV
-      - name: Run tests
+      - name: Run WPT tests
+        if: ${{ matrix.os == 'ubuntu-latest' && matrix.head == 'headful' }}
         timeout-minutes: 20
         # For verbose logging, add --log-mach - --log-mach-level info
         run: >
-          ${{ matrix.headless_command_prefix }}
-          ./wpt/wpt run
-          --webdriver-binary runBiDiServer.sh
-          ${{ matrix.wpt_headless_argument }}
-          --binary "${{ env.chromium_binary }}"
-          --manifest MANIFEST.json
-          --metadata wpt-metadata/mapper/${{ matrix.head }}
-          --log-wptreport out/wptreport-mapper.json
-          --timeout-multiplier 8
-          chrome
-          webdriver/tests/bidi/
+          xvfb-run --auto-servernum
+          ./runWPT.sh webdriver/tests/bidi/
+        env:
+          CHROME_BIN: ${{ env.chromium_binary }}
+          HEADLESS: false
+          WPT_REPORT: out/wptreport.json
+      - name: Run WPT tests
```

Comment:
It's simpler to define this twice with an `if` than to create a matrix (as it used to be), IMHO.